doc/tutorials/install-webui.md

```diff
@@ -27,12 +27,12 @@ To build Elektra with the elektra-web tool:
 
 - Start an elektrad instance: `kdb run-elektrad`
 - Start the client: `kdb run-web`
-- You can now access the client on: [http://localhost:33334](http://localhost:33334)
+- You can now access the client on: `http://localhost:33334`
```

Comment:
Any specific reason why you do not like the links?

Reply:
the linkchecker build step fails

Reply:
localhost can be safely added as exception :wink:

Reply:
didn't know its possible to add exceptions

Reply:
tests/linkchecker.whitelist

Reply:
it was easy to find as soon as i knew it existed :) fixed it